wollok-tests/src/wollok/imports/packageB/file2.wlk

```diff
@@ -0,0 +1,5 @@
+import packageA.file1.x
+
+object y{
+
+}
```

Comment:
Is there any test for project-relative imports like


project
  src
      wollok.root
      afolder
            afile
      bfolder
            bfile

and afile imports

import bfolder.bfile 

?